### DISCUSSION TITLE
Older downloads are offline

### DISCUSSION THREAD

**Discussion by michaelnum** [sumatrapdfreader/sumatrapdf#3475] [General]
Hello,
the downloads older than version 3.2 are offline.
The version 3. 12 for windows XP mentioned on the download page also.
Beste regards
Michael

**Comment by GitHubRulesOK** [comment]
reported see https://github.com/sumatrapdfreader/sumatrapdf/issues/3474

**Comment by glenvaughan** [comment]
Long story, but I really need the installer for version 3.0. Any idea where I can find it?

**Reply by GitHubRulesOK** [reply]
as per above link I can only suggest that you can virus total the offering from https://www.filehorse.com/download-sumatra-pdf-32/18339/ beware you avoid the ads etc and their copy seems good on virustotal shows as `c6e6db2b5d0b62210eafb5eee10f41d936d91809c00cbe2ea020de684443512c` however if your reason is printing there are better modern offerings apart from SumatraPDF from almost 10 years ago.

**Reply by glenvaughan** [reply]
Forgot to say it at the time, but thank you! This worked and was exactly what I needed. 

Nothing to do with printing, it's for an old, brittle Python script that requires copying the contents of a PDF and pasting them to a txt file before running. With every other PDF reader I've tried (including later versions of Sumatra), the formatting of the pasted results end up being different enough that it breaks the script. Hoping to phase out the script completely by the end of the year, and in the meantime it's much easier to just keep using the Sumatra version that works than to rewrite the script.

**Reply by GitHubRulesOK** [reply]
The best way to extract a page of text as per layout is using pdftotext (64bit 23-07 binary at https://github.com/oschwartz10612/poppler-windows/releases/ ) use the -layout option as all or one page or area of interest is best I know for simple scripts. or there is PyMuPDF ?

**Reply by dungsaga** [reply]
I have saved some old releases. You can get version 3.0 here: https://github.com/dungsaga/sumatrapdf-old-releases/blob/main/releases/SumatraPDF-3.0.zip

**Comment by kjk** [comment]
Sorry it took so long. It should work now.
